Manual/HTML/Lying Wires.html

```diff
@@ -0,0 +1,399 @@
+    <script src="js/ktane-utils.js"></script>
+    <script src="js/jquery.3.7.0.min.js"></script>
+    <script src="js/Utilities/svg-utils.js"></script>
+    <style>
+        .black {
+            background-color: black;
+            color: white;
+        }
+
+        .red {
+            background-color: red;
+            color: black;
+        }
+
+        .orange {
+            background-color: orange;
+            color: black;
+        }
+
+        .yellow {
+            background-color: yellow;
+            color: black;
+        }
+
+        .green {
+            background-color: green;
+            color: black;
+        }
+
+        .lime {
+            background-color: lime;
+            color: black;
+        }
+
+        .cyan {
+            background-color: cyan;
+            color: black;
+        }
+
+        .blue {
+            background-color: blue;
+            color: white;
+        }
+
+        .purple {
+            background-color: purple;
+            color: white;
+        }
+
+        .pink {
+            background-color: #FF3CC8;
+            color: black;
+        }
+
+        .white {
+            background-color: white;
+            color: black;
+        }
+
+        .grey {
+            background-color: grey;
+            color: black
+        }
+
+        .dark table,
+        table {
+            margin: 1em auto;
+            border: none;
+        }
+
+        .page-footer::before {
+            content: 'Page ';
+        }
+
+        .page-footer::after {
+            content: ' of 4';
+        }
+    </style>
+</head>
+
+<body>
+    <div class="section">
+        <div class="page page-bg-01">
+            <div class="page-header">
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">The Cruel Modkit</span>
+            </div>
+            <div class="page-content">
+                <h2>On the Subject of Lying Wires</h2>
+                <p class="flavour-text">One tells the truth… or two?… or three???</p>
+                <p><strong>Note: </strong>If your module’s components aren’t exactly <strong>[Wires, Button]</strong>,
+                    you are looking at the wrong section of this manual!</p>
+                <p>Tonight you’ll be playing a game of Lying Wires: every wire will give a True/False statement about
+                    the button, but some wires will lie about their statement depending on their star and LED colors.
+                    You must cut all the wires so that only the wires telling True statements are left, and then hold
+                    and release or tap the button to solve the game and the module. The host of this game can be found
+                    on the identity card. Refer to <b>Appendix: ID3NT1TYC4T3GORY</b> on the last page of this sub-module
+                    to
+                    identify which category your host belongs to.</p>
```

Comment:
This is more of a style concern that should probably be discussed with everyone involved on the Discord but I feel like appendices within a submodule is not necessary. In this manual, the appendix can be replaced with a table at the end of the submodule's manual with a relevant title